br/backup-and-restore-storages.md

```diff
@@ -1,247 +1,229 @@
 
-### S3 URL parameters
+**Back up snapshot data to GCS**
 
-| URL parameter | Description |
-|:----------|:---------|
-| `access-key` | The access key |
-| `secret-access-key` | The secret access key |
-| `use-accelerate-endpoint` | Whether to use the accelerate endpoint on Amazon S3 (default to `false`) |
-| `endpoint` | URL of custom endpoint for S3-compatible services (for example, `https://s3.example.com/`) |
-| `force-path-style` | Use path style access rather than virtual hosted style access (default to `true`) |
-| `storage-class` | Storage class of the uploaded objects (for example, `STANDARD`, `STANDARD_IA`) |
-| `sse` | Server-side encryption algorithm used to encrypt the upload (empty, `AES256` or `aws:kms`) |
-| `sse-kms-key-id` | If `sse` is set to `aws:kms`, specifies the KMS ID |
-| `acl` | Canned ACL of the uploaded objects (for example, `private`, `authenticated-read`) |
-| `role-arn` | When you need to access Amazon S3 data from a third party using a specified [IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html), you can specify the corresponding [Amazon Resource Name (ARN)](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the IAM role with the `role-arn` URL query parameter, such as `arn:aws:iam::888888888888:role/my-role`. For more information about using an IAM role to access Amazon S3 data from a third party, see [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html). |
-| `external-id` | When you access Amazon S3 data from a third party, you might need to specify a correct [external ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html) to assume [the IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html). In this case, you can use this `external-id` URL query parameter to specify the external ID. An external ID is an arbitrary string provided by the third party together with the IAM role ARN to access the Amazon S3 data. Providing an external ID is optional when assuming an IAM role, which means if the third party does not specify an external ID for the IAM role, you can assume the role and access the corresponding S3 data without providing this parameter. |
+```shell
+./br backup full --pd "${PD_IP}:2379" \
+--storage 'gcs://external/backup-20220915?credentials-file=${credentials-file-path}'
+```
 
-> **Note:**
->
-> It is not recommended to pass in the access key and secret access key directly in the storage URL, because these keys are logged in plain text.
+**Restore snapshot data from GCS**
 
-If neither the access key and secret access key nor IAM role ARN (`role-arn`) and external ID (`external-id`) are provided, the migration tools try to infer these keys from the environment in the following order:
+```shell
+./br restore full --pd "${PD_IP}:2379" \
+--storage 'gcs://external/backup-20220915?credentials-file=${credentials-file-path}'
```

Comment:
we should unify them `${secret_access_key}`, `${credentials-file-path}`, `${account key}`